.github/workflows/gke.yaml

```diff
@@ -1,29 +1,46 @@
-name: ConformanceGKE
+name: Trigger GKE connectivity test (see ConformanceGKE for results)
 
 on:
   pull_request_target:
```

Comment:
Since this is using `pull_request_target` please pin all GH actions to a specific commit sha, similarly to what is done here https://github.com/cilium/cilium/blob/c87bf30a042cf6e2e3efeebcc59adaf1dcf75276/.github/workflows/images-legacy.yaml#L55

Reply:
Done!